guides/common/modules/proc_importing-all-available-ansible-playbooks.adoc

```diff
@@ -0,0 +1,17 @@
+
+[id="importing-all-available-ansible-playbooks_{context}"]
+= Importing All Available Ansible Playbooks
+
+You can import all the available Ansible playbooks to {Project} from collections installed on {SmartProxy}.
+
+.Prerequisite
+* Ansible plug-in in {Project} is enabled
+
+.Procedure
+. Import the Ansible playbooks using the following API request:
```

Comment:
I was wondering: isn't this available in [hammer-cli-foreman-ansible](https://github.com/theforeman/hammer-cli-foreman-ansible). Looking at the source I only see one hit on `sync` (https://github.com/theforeman/hammer-cli-foreman-ansible/search?q=sync) and that's in https://github.com/theforeman/hammer-cli-foreman-ansible/blob/a2a884f1799e393f8a4ea340bc60c48dd8e2acc1/lib/hammer_cli_foreman_ansible/ansible_roles.rb#L28-L35

Given the filename and class name I get the impression it's only for roles, not playbooks. If we don't have this functionality, I think that deserves an issue to add it. Providing a CLI command is a better experience for most users than calling the API.

Reply:
AFAIK, it's only available via API, not other UIs yet.

Reply:
Bonus points, but in code I like to add comments for that. Something like
```suggestion
// Not available via Hammer/UI: <link-to-issue>
. Import the Ansible playbooks using the following API request:
```

Then in the future if you're reading it you know why it's done via the API and have an easy way to check if it's been resolved yet.

Reply:
> Then in the future if you're reading it you know why it's done via the API and have an easy way to check if it's been resolved yet.

Yeah, good idea